docs/howto/operator/gcp/bigquery.rst

```diff
@@ -255,32 +255,18 @@ Let's say you would like to execute the following query.
     :end-before: [END howto_operator_bigquery_query]
 
 To execute the SQL query in a specific BigQuery database you can use
-:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryExecuteQueryOperator`.
+:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryInsertJobOperator` with
+proper query job configuration.
 
 .. exampleinclude:: ../../../../airflow/providers/google/cloud/example_dags/example_bigquery_queries.py
     :language: python
     :dedent: 4
-    :start-after: [START howto_operator_bigquery_execute_query]
-    :end-before: [END howto_operator_bigquery_execute_query]
-
-``sql`` argument can receive a str representing a sql statement, a list of str
-(sql statements), or reference to a template file. Template reference are recognized
-by str ending in '.sql'.
+    :start-after: [START howto_operator_bigquery_insert_job]
+    :end-before: [END howto_operator_bigquery_insert_job]
```

Comment:
I think it woudl be nice to explain a bit here the job_id behavior -and how to make a job that is an "idempotent one". Just a sentence or two.

Reply:
Sure, added 👌